yt_dlp/aes.py

```diff
@@ -233,6 +232,8 @@ def aes_decrypt_text(data, password, key_size_bytes):
     @param {int} key_size_bytes          Possible values: 16 for 128-Bit, 24 for 192-Bit or 32 for 256-Bit
     @returns {str}                       Decrypted data
     """
+    from .compat import compat_b64decode
+
     NONCE_LENGTH_BYTES = 8
 
     data = bytes_to_intlist(compat_b64decode(data))
```

Comment:
just do `base64.b64decode`

Reply:
I'm moving them into local import, `utils` import in `compat` as well,